package.json

```diff
@@ -9,7 +9,7 @@
     "build": "gulp build",
     "dev": "gulp dev",
     "lint": "gulp lint",
-    "test": "gulp test",
+    "test": "gulp test --browsers PhantomJS",
```

Comment:
There didn't seem to be any way to overwrite the test command in Travis but I didn't think this is such a bad idea anyway.